<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pyBomQuote</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QFrame" name="frmCellInfo">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPlainTextEdit" name="txtCellInfo">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="backgroundVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QLabel" name="lblqty">
             <property name="text">
              <string>Quantity</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spnQty">
             <property name="maximumSize">
              <size>
               <width>16777208</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btnSetQty">
             <property name="text">
              <string>set quantity</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="edtSearch"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearchNext">
        <property name="text">
         <string>Search next</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnExportCarts">
        <property name="text">
         <string>export carts</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeBOM">
      <property name="columnCount">
       <number>4</number>
      </property>
      <column>
       <property name="text">
        <string notr="true">BOMPart/Supplier</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">Ordercode</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">Reference/Pricing</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">Description</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBOM">
    <property name="title">
     <string>BOM</string>
    </property>
    <addaction name="actionQuote_bom_into_file"/>
   </widget>
   <widget class="QMenu" name="menuQuote">
    <property name="title">
     <string>Quote</string>
    </property>
    <addaction name="actionOpen_quote_file"/>
    <addaction name="actionMultiply_part_quantity"/>
    <addaction name="actionFind_and_merge_duplicates"/>
    <addaction name="actionAdd_number_to_quantity_of_parts"/>
   </widget>
   <addaction name="menuBOM"/>
   <addaction name="menuQuote"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuote_bom_into_file">
   <property name="text">
    <string>Quote bom into quote file</string>
   </property>
  </action>
  <action name="actionOpen_quote_file">
   <property name="text">
    <string>Open quote file</string>
   </property>
  </action>
  <action name="actionMultiply_part_quantity">
   <property name="text">
    <string>Multiply part quantity</string>
   </property>
  </action>
  <action name="actionFind_and_merge_duplicates">
   <property name="text">
    <string>Find and merge duplicates</string>
   </property>
  </action>
  <action name="actionAdd_number_to_quantity_of_parts">
   <property name="text">
    <string>Add number to quantity of cheap parts</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
